src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+
+.stars {
+  display: inline-flex;
+  &__star {
+    align-items: center;
+    width: 16px;
+    height: 16px;
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+  @for $index from 1 through 5 {
+    &--#{$index} &__star:nth-child(-n + #{$index}) {
+      background-image: url("../images/star-active.svg");
+    }
+  }
+  &--4 .stars__star:not(:last-child) {
+    margin-right: 4px;
+  }
+}
```

Comment:
Add possibility to change amount of active stars by changing modifier(you could use styles from `product card` task).

And try to use cycle for
more detail in [video](https://mate.academy/learn/css-advanced/sass#/video/926)